### ISSUE TITLE
Enhancement: “gold-based decision-making”

### ISSUE THREAD

**Issue by Kiruscha** [Detergent13/tft-bot#76]
Hello,
here is another enhancement suggestion from my side:
Implement “gold detector” which can be used to make gold-dependent decisions, namely “rerolling” and the not-implemented feature of “buy xp”. 

Worst case, it would be just scanning for the gold numbers 50, 51, 52, 53, 54, 55, 56, 57… or so. Dunno I am not a software developer :>

**Comment by kizxlove** [comment]
I dont think this feature can work as you want because if you want to decide based on your gold, you must have a feature to track when it changes, which can't be working by pyautogui only. If you want to add this feature, you need to capture all gold numbers images from 01-50 and have a code for each of them.

**Comment by Detergent13** [comment]
Agreed, this would be super inefficient given the current libraries. (And wouldn't even be able to cover for every gold amount, since it could always grow past however far we capture.) 

In theory, this could be done with an OCR or text recognition library, but personally, that's probably out of my wheelhouse and quite a bit of work for something relatively small.

As it stands, I'm not too worried about making a particularly 'smart' bot- I'm just using this to farm tokens, and it's doing a pretty good job thus far. (Albeit in super low MMR with mostly other bots lol)
